1-data-marketplace/README.md

```diff
@@ -1 +1,81 @@
+| Ama.Analytics | useExistingSynapse |  Existing Purview account name | true or false |
+| Ama.Analytics | existingSynapseWorkspaceResourceId |
+| Ama.Analytics | useExistingPurview |  | true or false |
+| Ama.Analytics | purviewName | Existing Purview account name| If useExistingPurview=true, add manually the purviewName, else skip field |
+| Ama.Analytics | purviewResourceId | Existing Purview account ID | If useExistingPurview=true, add manually the purviewResourceID, else skip field |
+| Ama.Analytics | identity (objectId/clientId and clientSecret) |
+| Ama.Analytics | synapseSqlAdminGroupObjectId | Entra ID Group for Synapse Admin Group | Do not fill. Will be automatically generated by the prerequisites.sh script |
+| Ama | subOwner (objectId/clientId and clientSecret) | Identity used for the Analytics capacity | Do not fill. Will be automatically generated by the prerequisites.sh script |
+| Ama | environment |  Environment | Testing or Production |
+| Ama | managedResourceGroupName |  Resource group name for managed resources | DataMarketPlace_Internal_AMA_Mng |
+| Ama | offerTier |  Tier offering | standard |
+| Ama | crossTenant |  Data Sharing solution to work across tenants | tue or false |
+| Ama | name |  Name of the Managed application | E.g: DataMarketPlace_Internal |
+| Ama | tenantId |  Tenant ID | Fill the Tenant ID of the subscription |
+
+The prerequisites.sh script is responsible for the automation of:
+
+- 4 Resource Groups as defined in the config file (Definition RG, Managed App RG, Managed RG and Co-Manged RG)
+- 3 Entra ID groups as defined in the config file and replace just-in-time created Entra IDs in the config file
+- 2 Application Registrations in Entra ID: analytics-sp and subowner_sp replace just-in-time created Entra IDs in the config file
+- Signed-in user will be added to the KeyVault Entra ID and DevOps Entra ID groups
+- analytics-sp app registration will be added to the synapseSqlAdmin Entra ID Group
+- subowner-sp app registration will be added to the synapseSqlAdmin Entra ID Group
+- Reader RBAC role to analytics-sp
+- Owner Subscription RBAC role to subowner-sp
+
+To run the script, run the following commands:
+
+```bash
+cd 1-data-marketplace/scripts
+bash prerequirements.sh
+```
+
+### Graph Permissions
+
+Additionally, the analytics-sp needs to be granted to the following Graph API application permissions.
+In order to do that, manually give the sp_graph service principal created in the prerequirements.sh step General Administrator in your Entra ID.
+
+In order tu run the automation for the Graph permissions follow the steps below:
+
+1 - Navigate to 1-data-marketpace/scripts
+2 - Run pip install -r requirements.txt OR conda env create -f requirementmsgraphpython.yml
+3 - Run add-graph-permissions.py. This script will be responsible to add the following permissions to the sp_graph service principal:
+    - Applications.Read.All
+    - Group.Read.All
```

Comment:
```suggestion

    - Group.Read.All
    
```

Reply:
I get linting errors if I add the lines in between - can't change this one.